events/importer/enkora.py

```diff
@@ -1784,16 +1809,28 @@ def keyword_ids_into_keywords(kw_ids: set) -> list[Keyword]:
 
         return kws
 
-    def convert_title(self, course: dict) -> tuple[str, str]:
+    @staticmethod
+    def convert_title(course: dict) -> tuple[dict, Optional[dict]]:
         """
         Add information to course title about event hour and weekday
         :param course: dict, Enkora course information as returned from API
-        :return: tuple[str, str], modified course title
+        :return: tuple[dict, Optional[dict]], dictionary of course titles
+        and modified course titles in supported languages
         """
-        course_title = course["reservation_event_group_name"].strip()
+        course_title = {}
+
+        for lang in EnkoraImporter.supported_languages:
+            course_title.update(
+                {lang: course.get(f"reservation_event_group_name_{lang}")}
+            )
+
+        # If there's no course title found for key: reservation_event_group_name_fi, set the default one
+        if not course_title["fi"]:
+            course_title.update({"fi": course.get("reservation_event_group_name")})
```

Comment:
No need to use `dict.update()` in these and it makes them a bit harder to read IMO.
```suggestion
            course_title["fi"] = course.get("reservation_event_group_name")
```

Reply:
done